.github/workflows/update-google-fonts.yml

```diff
@@ -0,0 +1,25 @@
+name: Update Dependencies
+on:
+  schedule:
+    - cron:  '0 12 * * 1'
+jobs:
+  update-deps:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - name: Update list of Google fonts
+        id: vars
+        run: |
+          npm install
+          npm run download-fonts
```

Comment:
The `$GOOGLE_FONTS_API_KEY` env var needed for this has been added as a secret to this repository already.